.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+offerings:
+  - name: Retrieval_Augmented_Generation_Pattern
+    kind: solution
+    catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
+    offering_id: 5fdd0045-30fc-4013-a8bc-6db9d5447a52
+    variations:
+      - name: quickstart # Quick start with sample application (Deploy on Code Engine)
+        mark_ready: false
```

Comment:
its ok to set this to true - its set to false in the stacks repo because there is no upgrade test for stacks, so usually requires manually testing before publish